contribs/gmf/less/layertree.less

```diff
@@ -75,8 +51,15 @@
 
   .leaf,
   .group {
-    font-size: @font-size-small;
+    font-size: ceil(@font-size-base * 0.9);
```

Comment:
From where does the `font-size-base` come from? from bootstrap?
Doesn't is just be equals to 0.9rem?